app/com/linkedin/drelephant/tuning/AutoTuningAPIHelper.java

```diff
@@ -391,7 +396,7 @@ private JobExecution getJobExecution(TuningInput tuningInput) {
     logger.debug("Number of output parameters for execution " + tuningInput.getJobExecId() + " = "
         + jobSuggestedParamValues.size());
     logger.info("Finishing getCurrentRunParameters");
-   // return jobSuggestedParamValueListToMap(jobSuggestedParamValues);
+    // return jobSuggestedParamValueListToMap(jobSuggestedParamValues);
```

Comment:
Nit: Not related to this PR but remove commented out code

Reply:
ok